### PR TITLE
Implement enroll method and interface

### PR DESCRIPTION
**Pull request by realdevvora**

### CODE REVIEW

File: app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/course_class_imports_implementations/ArtsAndSciencesCourseInterface.java [hunk 1]
```diff
@@ -1,4 +1,8 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.interfaces.course_class_imports_implementations;
 
+import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.Course;
+import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.User;
+
 public interface ArtsAndSciencesCourseInterface {
+    public boolean enroll(User user);
```
Comment (uyoyo0) [inline app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/course_class_imports_implementations/ArtsAndSciencesCourseInterface.java:7]: Update the design doc to reflect the fact that you've added this method to the interface

File: app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/classes/course_subclasses/ArtsAndSciencesCourse.java [hunk 2]
```diff
@@ -1,7 +1,17 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.classes.course_subclasses;
 
 import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.Course;
+import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.User;
 
 public class ArtsAndSciencesCourse extends Course {
```
Comment (uyoyo0) [inline app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/classes/course_subclasses/ArtsAndSciencesCourse.java:6]: Looks good

**Review by uyoyo0 — DISMISSED** [review]

**Review by JaspreetKhela — CHANGES_REQUESTED** [review]
Please add comments to your code; you added some but try to add comments to the method signatures as well. Everything else looks good.

**Review by GagandeepLubana — APPROVED** [review]

**Review by JaspreetKhela — APPROVED** [review]

**Review by uyoyo0 — APPROVED** [review]
Looks even better with the comments

**Review by GagandeepLubana — APPROVED** [review]

**Pull request merged**